web/views/pages/index.ejs

```diff
@@ -39,14 +39,17 @@
         <!-- Pre-connect to improve performances -->
         <link rel="preconnect" href="https://api.electricitymap.org/">
```

Comment:
On a slightly off topic note, is this URL actually used anywhere in the frontend?
I can't find a reference to it or see any requests to it when using dev tools to inspect the webpage (both locally and public version)

Reply:
You're right, it's not - it should be changed to `https://app-backend.electricitymap.org` :)